src/apps/FEMSWEEP.cpp

```diff
@@ -0,0 +1,146 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-24, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "FEMSWEEP.hpp"
+#include "FEMSWEEP_DATA.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#include "common/DataUtils.hpp"
+
+#include <algorithm>
+#include <cmath>
+
+namespace rajaperf
+{
+namespace apps
+{
+
+
+FEMSWEEP::FEMSWEEP(const RunParams& params)
+  : KernelBase(rajaperf::Apps_FEMSWEEP, params)
+{
+  m_ne = 15 * 15 * 15;
+  m_na = 72;
+  m_ng = 128;
+
+  setDefaultProblemSize(ND * m_ne * m_ng * m_na);
+  setDefaultReps(1);
+
+  m_Blen = ND * m_ne * m_na;
+  m_Alen = ND * ND * m_ne * m_na;
+  // 9450 is a property of the mesh. Will need to derive this when mesh generator is available.
+  m_Flen = FDS * FDS * 2 * 9450 * m_na;
```

Comment:
Can we name these magic numbers and put them in the mesh description file?

Reply:
The magic numbers come from the generated mesh, and they vary depending on the dimensions of the mesh. I tried my best to make sense of them from the MFEM code and assign as many names as I could, but the remaining magic numbers are a mystery to me; I wouldn't even know what to call them because I can't determine what they represent. Let's leave them the way they are, and when we add a mesh generator to this kernel, we can update these to variable names which make sense.